harness/determined/pytorch/deepspeed/_deepspeed_trial.py

```diff
@@ -302,14 +302,11 @@ def _run(self) -> None:
         if self._dsat_mode:
             ops = self.context._core.searcher.operations()
             op = next(ops)
-            # TODO: read out the actual number of expected batches to run for from the config.
+            # TODO: remove `while True` in favor of exact number of steps which should run.
+            # TODO: also assuming here that op.length is always the number of *steps* to train for
+            # and never epochs or other units. Double check.
```

Comment:
Do we have control over that in our example configs? I don't fully understand the ramifications of what we mean by the operations length, especially in the context of our DS AT custom searchers

Reply:
Sorry, control over what? Over the units?